doc/en/user/source/extensions/geopkg-output/index.rst

```diff
@@ -0,0 +1,12 @@
+.. _geopkgoutput:
+
+GeoPackage Output
+=================
+
+This extension supports GeoPKG as an output format for both WFS (vectors) and WMS (tiles).
```

Comment:
The community module documentation does not seem to have been updated, should only report the WPS output format after this change, no?
https://docs.geoserver.org/latest/en/user/community/geopkg/index.html
and
https://docs.geoserver.org/latest/en/user/community/geopkg/output.html